docs/release-notes/2.43.0.md

```diff
@@ -0,0 +1,210 @@
+
+You can find the updated dashboards in the **MongoDB** and **Dashboards** sections of the main menu. They replace the previous versions as the default dashboards for MongoDB monitoring.
+
+The old dashboards have been moved to the **Experimental** folder and renamed with **(Old)** appended to their names. Future releases will remove these older versions.
+
+#### Updated MongoDB menu structure
+
+To complement the dashboard improvements, we've also restructured the MongoDB section on the main menu for better navigation and accessibility:
+
+- Reorganized menu for a more intuitive navigation, prioritizing the two updated GA dashboards above.
+- Decluttered view by moving older dashboards to the **Experimental** folder or removing them from the menu.
+- Added **MongoDB Oplog Details**, **MongoDB Collection Overview**, and **MongoDB Instance Summary** to the main menu.
+  
+    ![New MongoDB menu structure](../_images/new_Mongo_menu.png)
+
+We encourage you to start using the new dashboards to benefit from their enhanced monitoring capabilities. We also invite you to share your feedback in the [PMM forum](https://forums.percona.com/c/percona-monitoring-and-management-pmm/percona-monitoring-and-management-pmm-v2/31).
+
+### New MongoDB collector: CurrentOp
+
+The MongoDB exporter now includes the **CurrentOp** collector, offering visibility into active operations, including the new `mongodb_currentop_query_uptime` metric.
+
+To start the MongoDB exporter with the CurrentOp collector enabled, use the `--enable-all-collectors` flag when adding MongoDB services to your PMM instance:
+
+```sh
+pmm-admin add mongodb --enable-all-collectors ...
+```
+
+Specifying a limit with `--max-collections` for this collector is not necessary.
+
+!!! note "Operation collection limitation"
+    To minimize impact on disk usage, the CurrentOp collector is designed to collect only operations that have been running for longer than 1 minute. This limitation helps focus on potentially problematic long-running operations while keeping data volume manageable.
+
+For more information on MongoDB collectors and metrics, see the [pmm-admin commands documentation](../details/commands/pmm-admin.md).
+
+### Monitoring for Percona Backup for MongoDB
+
+PMM now includes support for monitoring your Percona Backup for MongoDB (PBM) setup, providing valuable insights with metrics related to `pbm-agent` statuses, PITR (Point-In-Time Recovery) configuration, and backup statuses.
+
+This new functionality is enabled through a dedicated PBM collector within the MongoDB exporter, allowing you to easily monitor your PBM setup with the following metrics:
+
+- `mongodb_pbm_cluster_backup_configured`
+- `mongodb_pbm_agent_status`
+- `mongodb_pbm_cluster_pitr_backup_enabled`
+- `mongodb_pbm_backup_size`
+- `mongodb_pbm_backup_duration_seconds`
+
+These metrics are enabled by default. If you’re already using PBM, you’ll automatically start receiving these metrics without additional configuration.
```

Comment:
is there a dashboard to see this info?  or is it integrated with the backup interface? can I create alerts off of them?  so many questions!

Reply:
Hi Steve, these metrics can be seen on the Explore tab for now.

We have a couple of alert templates planned based off them (such as alerting when pbm agent is down), but this one was about getting the underlying work of collecting those metrics out of the door.